docs/Reference.md

```diff
@@ -253,24 +253,27 @@ Discord utilizes a subset of markdown for rendering message content on its clien
 
 ###### Formats
 
-| Type                    | Structure           | Example                      |
-| ----------------------- | ------------------- | ---------------------------- |
-| User                    | <@USER_ID>          | <@80351110224678912>         |
-| User \*                 | <@!USER_ID>         | <@!80351110224678912>        |
-| Channel                 | <#CHANNEL_ID>       | <#103735883630395392>        |
-| Role                    | <@&ROLE_ID>         | <@&165511591545143296>       |
-| Standard Emoji          | Unicode Characters  | 💯                            |
-| Custom Emoji            | <:NAME:ID>          | <:mmLol:216154654256398347>  |
-| Custom Emoji (Animated) | <a:NAME:ID>         | <a:b1nzy:392938283556143104> |
-| Unix Timestamp          | <t:TIMESTAMP>       | <t:1618953630>               |
-| Unix Timestamp (Styled) | <t:TIMESTAMP:STYLE> | <t:1618953630:d>             |
+| Type                     | Structure           | Example                       |
+| ------------------------ | ------------------- | ----------------------------- |
+| User                     | <@USER_ID>          | <@80351110224678912>          |
+| User \*                  | <@!USER_ID>         | <@!80351110224678912>         |
+| Channel                  | <#CHANNEL_ID>       | <#103735883630395392>         |
+| Role                     | <@&ROLE_ID>         | <@&165511591545143296>        |
+| Application Command \*\* | </NAME:ID>          | </airhorn:816437322781949972> |
```

Comment:
```suggestion
| Slash Command \*\*       | </NAME:COMMAND_ID>          | </airhorn:816437322781949972> |
```

Reply:
mentions are only available for slash commands since other application commands aren't global in the same way